examples/vertexai/server-tools.php

```diff
@@ -28,6 +25,6 @@
     ),
 );
 
-$result = $platform->invoke($model, $messages);
+$result = $platform->invoke('gemini-2.5-pro', $messages, ['server_tools' => ['url_context' => true]]);
```

Comment:
@chr-hertel I think it is fine to add the options via the platform, instead of the model here. Please resolve if you are fine.